go.mod

```diff
@@ -12,10 +12,10 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/improbable-eng/grpc-web v0.15.0
 	github.com/logrusorgru/aurora v2.0.3+incompatible
-	github.com/onflow/cadence v0.42.9
+	github.com/onflow/cadence v0.42.10
 	github.com/onflow/flow-core-contracts/lib/go/templates v0.15.1
 	github.com/onflow/flow-go v0.33.2-0.20240411115514-7b9997bb6c14
```

Comment:
Maybe also update flow-go while at it. https://github.com/onflow/flow-go/pull/5654 is in now, so can update to [015156b](https://github.com/onflow/flow-go/commit/015156b297b5a7bbc3b7c3ce96739709633d0ac2).